src/main/resources/i18n/messages.properties

```diff
@@ -126,6 +126,17 @@ email.successfulDataExportCreationsAdmin.title = Successfully created requested
 email.successfulDataExportCreationsAdmin.text = Data exports for the following users were successfully created when the data export creation job was ran:
 email.successfulDataExportCreationsAdmin.userLogin = \u2022 {0}
 
+# SSH User Settings
+email.sshKeyAdded.title = New SSH key added to account
+email.notification.sshKeyAdded.title = The following SSH key was added to your account:
```

Comment:
Please make the first mail more consistent in regards to `:` and labeling of the information :)

Reply:
Changed it, to be like the other two mails, see the updated screenshot